### DISCUSSION TITLE
window11 打开一个窗口后再打开第二个会白屏

### DISCUSSION THREAD

**Discussion by shoebillback** [xiaoyaocz/flutter_dmzj#126] [General]
<img width="1280" alt="Snipaste_2023-03-17_20-58-49" src="https://user-images.githubusercontent.com/101227862/225911587-c80a4000-8b42-41d7-adee-9e63e871446c.png">
如图 版本v2.0.3

**Comment by xiaoyaocz** [comment]
Flutter暂时只支持单窗口 https://github.com/flutter/flutter/issues/30701

**Reply by shoebillback** [reply]
嗯, 我的意思是可以强制关闭掉第二个窗口之类的, 不过不是什么大问题

**Comment by xiaoyaocz** [comment]
#142
